upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -123,6 +123,9 @@ type ApplyClusterCmd struct {
 
 	// Phase can be set to a Phase to run the specific subset of tasks, if we don't want to run everything
 	Phase Phase
+
+	// LifecycleOverrides contains a map of lifecycle values to fi.Lifecycle
```

Comment:
This comment just restates the type signature.  Remove, or replace with something that describes what it is for (e.g. how does this interact with phases)